content/08.outlook.md

```diff
@@ -1 +1,41 @@
+We believe that the same challenges that are currently outlined as major pitfalls in applying ML to rare disease can be great opportunities for data generation as well as method development in moving the field forward.
+During our journey through the various challenges, we identified two major areas where mindful strategies can immeasurably enhance the power of machine learning in rare disease and move the field forward.
+
+_Emphasis on not just "more n" but "more meaningful n"._
+
+Mindful addition of data is key for powering the next generation of analysis in rare disease data.
+While there are many techniques to collate rare data from different sources, incorrect data generation may hurt the end goal even if it adds to the size of the dataset.
+In our experience collaboration with domain experts have proved to be critical in gaining insight into potential sources of variation in the datasets.
+As an example, in a study of neurofibromatosis type 1 (NF1) some samples were generated using vastly different surgical techniques (laser ablation and excision vs standard excision). [@doi:10.3390/genes11020226] 
+This resulted in surfacing of substantial biological differences that are a consequence of process (e.g. activation of heat shock protein related pathways), not disease related biology. 
+Prior collaboration with domain experts would have helped minimize these differences and generate a more robust dataset.
+A few such collaborations are beginning to form and show promise in generating valuable datasets for future use.[@https://www.hopkinsmedicine.org/kimmel_cancer_center/cancers_we_treat/pediatric_oncology/research_and_clinical_trials/pratilas/nf1_biospecimen_repository.html]
+
+Rare disease not only lacks samples, there is also a great dearth of comprehensive phenotypic-genotypic databases. 
+With the ubiquity of sequencing platforms, genomic data has been easy to gather for rare disease patients.
+This is augmented by the fact that most rare diseases seem to be genetic in origin.
+An important next step would be to integrate genomic and clinical data to produce comprehensive phenotypic-genotypic databases that can fuel interpretation of features extracted using ML methods.
+[TODO: add Refs]
+
+Finally, mindful sharing of data with proper metadata and attribution to enable prompt data reuse is of utmost important in building datasets that can be of great value in rare disease. [@https://www.nature.com/articles/s41576-020-0257-5]
+[TODO: expand on this]
+
+
+_Development of methods that reliably support mechanistic interrogation of specific rare diseases_
+
+A pattern that emerged readily in our literature search of application of ML methods in rare disease is that most ML related strategies were focused on classification of a patient as a rare disease patient among other non-rare disease patients.
```

Comment:
I am hesitant to talk about a literature search for "this is not a systematic review" reasons.